test/integration/elements/test-element.ts

```diff
@@ -18,9 +18,6 @@ class TestElement extends Polymer.Element {
   @property({notify: true})
   aNum: number = 42;
 
-  @property()
-  doesntNotify: boolean = true;
```

Comment:
there's a failing test causing your build to fail which depends on this. maybe just restore this property